Source/Core/DolphinQt/Achievements/AchievementHeaderWidget.cpp

```diff
@@ -0,0 +1,140 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#ifdef USE_RETRO_ACHIEVEMENTS
+#include "DolphinQt/Achievements/AchievementHeaderWidget.h"
+
+#include <QCheckBox>
+#include <QGroupBox>
+#include <QLabel>
+#include <QLineEdit>
+#include <QProgressBar>
+#include <QPushButton>
+#include <QString>
+#include <QVBoxLayout>
+
+#include <fmt/format.h>
+
+#include <rcheevos/include/rc_api_runtime.h>
+#include <rcheevos/include/rc_api_user.h>
+#include <rcheevos/include/rc_runtime.h>
+
+#include "Core/AchievementManager.h"
+#include "Core/Core.h"
+
+#include "DolphinQt/QtUtils/ModalMessageBox.h"
+#include "DolphinQt/QtUtils/NonDefaultQPushButton.h"
+#include "DolphinQt/QtUtils/SignalBlocking.h"
+#include "DolphinQt/Settings.h"
+
+AchievementHeaderWidget::AchievementHeaderWidget(QWidget* parent) : QWidget(parent)
+{
+  m_user_name = new QLabel();
+  m_user_points = new QLabel();
+  m_game_name = new QLabel();
+  m_game_points = new QLabel();
+  m_game_progress_hard = new QProgressBar();
+  m_game_progress_soft = new QProgressBar();
+  m_rich_presence = new QLabel();
+
+  QVBoxLayout* m_user_right_col = new QVBoxLayout();
+  m_user_right_col->addWidget(m_user_name);
+  m_user_right_col->addWidget(m_user_points);
+  QHBoxLayout* m_user_layout = new QHBoxLayout();
+  // TODO lillyjade 02 Jun 2023 - player badge goes here
```

Comment:
Please don't leave name and date here. Same for the other instances of that.

Reply:
Fair, just intended it as a note to myself so I could find it more easily in the future. Do you have any recommendations?